airbyte-integrations/connectors/source-chargebee/source_chargebee/manifest.yaml

```diff
@@ -35,8 +35,8 @@ definitions:
       datetime: "{{ format_datetime(config['start_date'], '%s') }}"
       datetime_format: "%s"
     end_datetime:
-      datetime: "{{ now_utc().strftime('%s') }}"
-      datetime_format: "%s"
+        datetime: "{{ now_utc().strftime('%Y-%m-%dT%H:%M:%SZ') }}"
```

Comment:
I got trolled by [that](https://github.com/airbytehq/airbyte-python-cdk/blob/e27cb81196fec42e2a17f6fc728c7f6f066be194/airbyte_cdk/sources/declarative/datetime/datetime_parser.py#L41). As this is not expose anywhere, I've decided to update the format to one that won't break with timezone differences.